src/raft_server.c

```diff
@@ -41,6 +41,8 @@ static void __log(raft_server_t *me_, raft_node_t* node, const char *fmt, ...)
     vsprintf(buf, fmt, args);
```

Comment:
may be overkill: but what do you think about changing the call to vsnprintf(buf, 1024, fmt, args) to avoid any (very unlikely) buffer overrun depending on what the caller provides for fmt and ... args?

Reply:
Good idea. Coverity should really catch this...